### DISCUSSION TITLE
Internal Server Error after modification of Jira

### DISCUSSION THREAD

**Discussion by ndenisof** [DefectDojo/django-DefectDojo#10169] [Q&A]
hi!
We have a jira instance that does not support basic auth. So I edited file jira_link/helper.py to get auth by bearer token.

But now I get a lot of errors like
Internal Server Error: /finding/3203
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/db/utils.py", line 98, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.InvalidCursorName: cursor "_django_curs_140605104562752_sync_45" does not exist

What should I consider trying to fix that? Should I edit something else?

**Comment by Tiarin-Hino** [comment]
Can you share code that you have changed? Hard to help without seeing what was changed. I had similar issue so might be able to help.
